osu.Game.Rulesets.Taiko.Tests/TestSceneTaikoHitObjectSamples.cs

```diff
@@ -17,7 +15,6 @@ public class TestSceneTaikoHitObjectSamples : HitObjectSampleTest
         protected override IResourceStore<byte[]> RulesetResources => new DllResourceStore(Assembly.GetAssembly(typeof(TestSceneTaikoHitObjectSamples)));
 
         [TestCase("taiko-normal-hitnormal")]
-        [TestCase("normal-hitnormal")]
         [TestCase("hitnormal")]
```

Comment:
This fallback shouldn't exist either, right?

Reply:
This one is more of a general fallback for all sample lookups on a legacy skin (not part of `LookupNames`), not sure if stable uses universal samples in taiko gameplay.